### DISCUSSION TITLE
First contribution - Track HTTP response status of each pageview

### DISCUSSION THREAD

**Discussion by eriknakata** [plausible/analytics#1549] [Anything Else]
Hello,

I trying to start the development of tracking the HTTP status code but have some problems.

First one: I imagine that the plausible javascript should send the status code posting the pageview on /api/event.

Example:

```json
{"n": "pageview", "u": "https://plausible.io/", "d": null, "r": null, "w": 1680, "s" 200}
```

The problem is: There is no API on Javascript side to get the last status code.

Second problem: If the server returns a 404, there is no guarantee the current HTML contains the plausible file. So the page view will not be triggered.

Any idea how to solve this? Is really possible?

Thanks in advance

**Comment by ukutaht** [comment]
You're right, I thought there would be an easy way to get the status code with Javascript but there isn't 🤷

I think we'll have to scratch the plan for this feature. We have a way to track 404 pages anyways. I thought  it would be possible to do it without any extra work but that doesn't seem to be the case :(
